content/docs/ops/setup/app-health-check/index.md

```diff
@@ -166,7 +114,18 @@ spec by yourself.
 The configuration changes above (by Helm or by the configuration map) effect all Istio app deployments.
 {{< /warning >}}
 
-##### Re-deploy the liveness health check app
+#### Use Annotations on Pod
+
+<!-- Add samples YAML or kubectl patch? -->
```

Comment:
I'm thinking to add sample YAML inlined here, as 

```yaml
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  name: liveness-http
spec:
  template:
    metadata:
      labels:
        app: liveness-http
        version: v1
      annotations:
        sidecar.istio.io/rewriteAppHTTPProbers: "true"
    spec:
      containers:
      - name: liveness-http
        image: docker.io/istio/health:example
        ports:
        - containerPort: 8001
        livenessProbe:
          httpGet:
            path: /foo
            port: 8001
          initialDelaySeconds: 5
          periodSeconds: 5
```

And let people to do kubectl apply, as opposed to just mention the annotation. I think that makes things easier to follow? wtdy? @rcaballeromx

Reply:
code snippet to copy is best imho

Reply:
Added.